CMakeLists.txt

```diff
@@ -254,8 +264,7 @@ endif()
 
 message(STATUS "Hip compiler flags: ${HIP_COMPILER_FLAGS}")
 
-add_definitions("-DHIP_COMPILER_FLAGS=${HIP_COMPILER_FLAGS}")
-
+add_compile_definitions($<$<COMPILE_LANGUAGE:CXX>:HIP_COMPILER_FLAGS="${HIP_COMPILER_FLAGS}">)
```

Comment:
@apwojcik 🐛 These extra double ticks break HIP kernel builds on offline path (and possibly on COMgr path), thus affecting Tuna. Fix TBD soon.

/cc @JehandadKhan @junliume

Reply:
Fixed in #2544